src/style.css

```diff
@@ -0,0 +1,159 @@
+/* root */
```

Comment:
remove comments

Reply:
will be better to create a separate file for reset

Reply:
also, check this checklist point 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/102411130/226281186-b50d0d40-4bb2-455b-8157-0fcc1b364069.png">